File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {

    public static void main(String[] args) throws IOException {
        solve();
    }

    private static void solve() throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        int [] array = new int [N];
        int count = 0;
        StringTokenizer st = new StringTokenizer(br.readLine());
        for(int i=0; i<N; i++) {
            array[i] = Integer.parseInt(st.nextToken());
        }
        int v = Integer.parseInt(br.readLine());
        for(int i=0; i<N; i++) {
            if(array[i] == v) {
                count++;
            }
        }
        System.out.println(count);
        br.close();
    }
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.kt
import java.util.StringTokenizer

fun main() {
    readln()
    var count: Int = 0
    val st = StringTokenizer(readln())
    val V = readln().toInt()
    while (st.hasMoreTokens()) {
        if (st.nextToken().toInt().equals(V)) {
            count++
        }
    }

    println(count)
}
